en/Plugins/Getting started/Mobile development.md

```diff
@@ -18,6 +18,17 @@ To disable mobile emulation, enter the following and press `Enter`:
 this.app.emulateMobile(false);
 ```
 
+## Inspecting the webview on the actual mobile device
```

Comment:
This is the wrong place to put it - the next section is still talking about mobile emulation on the desktop app.

I suggest moving this section to the very end of the document.

Reply:
Or at least just before "Platform-specific features"

Reply:
True I missed the callout there 😅